views/index.pug

```diff
@@ -20,14 +20,16 @@ block content
          .carousel-inner(role='listbox')
            each menu in menus
              .carousel-item.active
-               img(src= menu.image + '.jpg' width="600px" height="300px")
+               img(src= menu.image width="600px" height="300px")
                .carousel-caption
                  .row
                    .col-lg-6
                      a(href='/menus/' + menu.slug)
                        button.btn.btn-info(type='button') More info
                    .col-lg-6
-                     button.btn.btn-info(type='button') Vote
+                     form(action="/menus/" + menu.id + "/votes" method="POST")
+                       input(type="hidden" name="MenuId" value=menu.id)
```

Comment:
you don't need this hidden field if you are making the request to /menus/:menu_id/votes